system-tests/__snapshots__/record_spec.js

```diff
@@ -160,7 +160,7 @@ plugin stdout
   │ Pending:      1                                                                                │
   │ Skipped:      0                                                                                │
   │ Screenshots:  1                                                                                │
-  │ Video:        true                                                                             │
+  │ Video:        false                                                                            │
```

Comment:
snapshot is different because we are now deleting the video entirely, which is slightly different than `videoUploadOnPasses`, which keeps the video but skips compression and does NOT upload it to the cloud.